### PR TITLE
BAH-1938|Added config for crater-atomfeed and crater-atomfeed-db

### PR DESCRIPTION
**Pull request by stavtnameh**
Co-authored-by: Divij Goyal <divij.g@beehyv.com>

### CODE REVIEW

File: bahmni-docker/.env [hunk 1]
```diff
@@ -139,4 +139,5 @@ CRATER_ATOMFEED_DB_PASSWORD=beehyv123
 CRATER_ATOMFEED_DB_ROOT_PASSWORD=beehyv123
 CRATER_ATOMFEED_DB_HOST=crater-atomfeed-db
 CRATER_ATOMFEED_DB_PORT=3306
-CRATER_ATOMFEED_DB_NAME=crater_atomfeed
\ No newline at end of file
+CRATER_ATOMFEED_DB_NAME=crater_atomfeed
+CRATER_URL=https://demo.craterapp.com
```
Comment (gokultw) [inline bahmni-docker/.env:143]: this instance of the Crater app is not maintained by us.

File: bahmni-docker/.env [hunk 2]
```diff
@@ -130,3 +130,14 @@ PACS_SIMULATOR_TIMEOUT=20000
 PACS_SERVER_TYPE=dcm4chee
 PACS_SERVER_URL=DCM4CHEE@dcm4chee:11112
 UPDATE_PACS_INTEGRATION_DB=true
+
+#Crater-Atomfeed Environment variables
+CRATER_USERNAME=admin@craterapp.com
+CRATER_PASSWORD=crater@123
+CRATER_ATOMFEED_DB_USERNAME=beehyv
```
Comment (mohan-13) [inline bahmni-docker/.env:137]: Use some generic password.

File: bahmni-docker/.env (hunk 1 above)
Comment (mohan-13) [reply bahmni-docker/.env:143]: Yup. it should be localhost:81

**Review by mohan-13 — COMMENTED** [review]
Update `bahmni-docker` README at the necessary places.

**Review by gsluthra — APPROVED** [review]

File: bahmni-docker/.env [hunk 3]
```diff
@@ -130,3 +130,14 @@ PACS_SIMULATOR_TIMEOUT=20000
...
+CRATER_ATOMFEED_DB_PASSWORD=beehyv123
+CRATER_ATOMFEED_DB_ROOT_PASSWORD=beehyv123
+CRATER_ATOMFEED_DB_HOST=crater-atomfeed-db
+CRATER_ATOMFEED_DB_PORT=3306
+CRATER_ATOMFEED_DB_NAME=crater_atomfeed
+CRATER_URL=http://localhost:81
```
Comment (mohan-13) [inline bahmni-docker/.env:143]: @stavtnameh With this configuration is the atomfeed sync working with local openmrs and crater ? I feel that the URL would need a change. `localhost` would resolve within the container. Please test it.

File: bahmni-docker/.env (hunk 2 above)
Comment (mohan-13) [reply bahmni-docker/.env:137]: I feel that we should remove `beehyv` and use some common credentials

**Review by mohan-13 — APPROVED** [review]

**Pull request merged**